test/inputs/texts.js

```diff
@@ -64,3 +64,7 @@ export async function textReadonly() {
 export async function textPassword() {
   return Inputs.text({type: "password"});
 }
+
+export async function textDate() {
+  return Inputs.text({type: "date"});
```

Comment:
```suggestion
  return Inputs.text({type: "date", min: "2005-06-12", max: "2022-03-01"});
```

why not test min and max here?

Reply:
yes of course, good idea. Thank you